src/test/Makefile

```
@@ -481,8 +481,8 @@ GTEST_DIR = ../../lib/test/gtest
 
 # Use clang/clang++ by default
 
-CC  := clang-12
-CXX := clang++-12
+CC  := clang-15
+CXX := clang++-15
```

https://github.com/betaflight/betaflight/compare/master...ledvinap:betaflight:makefile-clang
```suggestion
CC  ?= clang-15
CXX ?= clang++-15
```

---

I merged this, but the build then fails, so I've backed it out.

---

maybe you have CC/CXX set in your environment? 
if it is macos thing, use :=

---

It failed the GitHub build, so not MacOS.